edx_adapt/tests/performance_tests.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from datetime import datetime, timedelta
+from selenium import webdriver
+from selenium.webdriver.support.events import EventFiringWebDriver
+from selenium.webdriver.support.events import AbstractEventListener
```

Comment:
We could leave only one import and use event.EventFiringWebDriver and event.AbstractEventListener in the code